community-operators/prometheus/prometheusoperator.0.27.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,295 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: prometheusoperator.0.27.0
+  namespace: placeholder
+  annotations:
+    categories: Monitoring
+    certified: "false"
+    description: Manage the full lifecycle of configuring and managing Prometheus and Alertmanager servers.
+    containerImage: quay.io/coreos/prometheus-operator:v0.27.0
+    createdAt: 2019-01-08T16:12:00Z
+    support: Frederic Branczyk
+    alm-examples: '[{"apiVersion":"monitoring.coreos.com/v1","kind":"Prometheus","metadata":{"name":"example","labels":{"prometheus":"k8s"}},"spec":{"replicas":2,"version":"v2.7.1","serviceAccountName":"prometheus-k8s","securityContext": {}, "serviceMonitorSelector":{},"ruleSelector":{},"alerting":{"alertmanagers":[{"namespace":"monitoring","name":"alertmanager-main","port":"web"}]}}},{"apiVersion":"monitoring.coreos.com/v1","kind":"ServiceMonitor","metadata":{"name":"example","labels":{"k8s-app":"prometheus"}},"spec":{"selector":{"matchLabels":{"k8s-app":"prometheus"}},"endpoints":[{"port":"web","interval":"30s"}]}},{"apiVersion":"monitoring.coreos.com/v1","kind":"Alertmanager","metadata":{"name":"alertmanager-main"},"spec":{"replicas":3, "securityContext": {}}}]'
+spec:
+  replaces: prometheusoperator.0.22.0
```

Comment:
this should be `prometheusoperator.0.22.2` so that it references the other CSV in the registry